Rakefile

```diff
@@ -0,0 +1,4 @@
+task :run_crawler do
+  puts "rake task running"
```

Comment:
Probably don't need this `puts`, or at least you could have something more descriptive here.